src/app/modules/main/chat_section/module.nim

```diff
@@ -1320,7 +1319,7 @@ method createOrEditCommunityTokenPermission*(self: Module, communityId: string,
       self.onCommunityTokenPermissionUpdateFailed(communityId)
       return
 
-    tokenCriteriaDto.amount = viewAmount.formatBiggestFloat(ffDecimal)
+    tokenCriteriaDto.amountInWeis = tokenCriteria{"amount"}.getStr
```

Comment:
we fill new field

Reply:
We can rename it `amountInWei` for it to sound better

Reply:
Like everywhere? 😱

Reply:
`CTRL+SHIFT+F` -> `Replace all` :grin: